### PR TITLE
Fix difference between selector labels in serviceMonitor and service

### PR DESCRIPTION
**Pull request by notariuss**
Current selector doesn't work: https://prnt.sc/spjczt

### CODE REVIEW

File: chart/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -15,10 +15,15 @@ metadata:
...
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    helm.sh/from: deploy.{{ include "app.fullname" . }}
+    matchLabels:
+      app.kubernetes.io/name: {{ include "app.name" . }}
+      app.kubernetes.io/version: {{ .Chart.AppVersion }}
+      helm.sh/chart: {{ include "app.chart" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/managed-by: {{ .Release.Service }}
+      helm.sh/from: deploy.{{ include "app.fullname" . }}
+      {{- toYaml .Values.service.labels | indent 6 }}
```
Comment (mvisonneau) [inline chart/templates/servicemonitor.yaml:25]: I think you need `nindent` in here as well

Comment (notariuss) [reply chart/templates/servicemonitor.yaml]: fixed

**Comment by mvisonneau** [comment]
cheers @notariuss 🙇

**Comment by notariuss** [comment]
It works!))
image

**Comment by mvisonneau** [comment]
awesome,  thanks for fixing this. I did not actually use it myself yet after having written it 🤦

**Pull request merged**